modules/videostab/include/opencv2/videostab/frame_source.hpp

```diff
@@ -86,6 +86,24 @@ class CV_EXPORTS VideoFileSource : public IFrameSource
     Ptr<IFrameSource> impl;
 };
 
+class CV_EXPORTS MaskFrameSource : public IFrameSource
```

Comment:
I would prefer to remove "CV_EXPORTS" from here and define all methods inline in the header file.

Reply:
One quick question, your recommendation breaks the style in frame_source.hpp. 
Currently it is consistent, i.e.
`class CV_EXPORTS NullFrameSource : public IFrameSource`
`class CV_EXPORTS VideoFileSource : public  IFrameSource`
`class CV_EXPORTS MaskFrameSource : public IFrameSource`

Reply:
These is no reason to make exportable API (like `NullFrameSource`) with very simple implementation.
`MaskFrameSource` looks like a proxy with "callback".

Reply:
Just wanted to point that out, I'll change this as recommended